src/foam/u2/view/LoginView.js

```diff
@@ -301,10 +304,14 @@ foam.CLASS({
         .addClass(self.myClass('content-form'))
         .callIf(self.displayWidth, function() { this.onDetach(self.displayWidth$.sub(self.resize)); })
         .startContext({ data: this }).tag(this.DATA).endContext()
-        .start()
-          .addClass('align-end')
-          .tag(this.data.SUB_FOOTER)
-        .end()
+        .callIfElse(this.data.subFooter, function() {
```

Comment:
am i able to check the isAvailable flag somehow? So i don't need to remove the subFooter property from SignUp.js model

Reply:
@sarthak-marwaha ^